modules/administration-guide/partials/proc_restoring-a-che-instance-from-a-backup-by-using-the-checlusterrestore-object.adoc

```diff
@@ -0,0 +1,64 @@
+[id="restoring-a-{prod-id-short}-instance-from-a-backup-by-using-the-checlusterrestore-object_{context}"]
+= Restoring a {prod-short} instance from a backup by using the CheClusterRestore custom object
+
+You can restore a {prod-short} instance from a backup snapshot by using custom resources. A new `CheClusterRestore` custom object is required every time you restore a {prod-short} instance from a backup snapshot; that is, editing an already consumed `CheClusterRestore` custom object, during or after backing up, has no effect.
```

Comment:
```suggestion
You can restore a {prod-short} instance from a backup snapshot by using custom resources. A new `CheClusterRestore` custom object is required every time you restore a {prod-short} instance from a backup snapshot; editing an already consumed `CheClusterRestore` custom object, during or after backing up, has no effect.
```
"that is" is a nice structure but can be considered redundant in our dry and concise tech docs.

Reply:
@missmesss, thank you :) This is how I've reworked this:

![image](https://user-images.githubusercontent.com/78564304/147366051-d41a90dd-4578-42e9-ba5a-3623c4a98a6c.png)